UPGRADING.md

```diff
@@ -2,6 +2,11 @@
 
 This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
 
+## v3
+
+* Dropped support for node 4
+* Batch operations no longer default to `.put`
```

Comment:
Maybe add "If no `type` is specified, an error will be thrown, courtesy of `abstract-leveldown`."

Reply:
Don't want this to hold off release though, rest LGTM!